### ISSUE TITLE
Other Berean Translations?

### ISSUE THREAD

**Issue by Jab2870** [gracious-tech/paper#2]
I love the BSB which is obviously the default in paper, however I'd love to see the interlinear and literal versions added.

**Comment by shadow-light** [comment]
Yes, as soon as they release USFM for them we'll add them. They are still in draft I think.

**Comment by Gajusbonus** [comment]
Is it possible to print the LOGOS bible? Translated from Septuagint, with majority text.

**Comment by Gajusbonus** [comment]
> Is it possible to print the LOGOS bible? Translated from Septuagint, with majority text.

Can you please answer how to add a bible? Can I help adding it?

**Comment by shadow-light** [comment]
Sorry, forgot to reply. Only USFM or USX is accepted at the moment. Though it's possible to convert to them. I'd need a link to the source of the Bible translation you're interested in.

**Comment by Gajusbonus** [comment]
The link to LOGOS bible is:
https://mysword-bible.info/download/getfile.php?file=logos.bbl.mybible.gz

**Comment by shadow-light** [comment]
I couldn't find a license for that translation so I assume it is under full copyright. I'm only adding freely shareable translations, but if you'd like to use it yourself you can create your own collection and still use the fetch(bible) platform. https://fetch.bible/access/collections/

**Comment by Gajusbonus** [comment]
I understand, good point.
Now I found a Bible which is in the public domain. it called "Majority Standard Bible" and you can find it here in various formats. https://majoritybible.com
The OT is same as BSB, the NT is Majority Text.
 I'm looking forward to see this on paper :-0

**Comment by shadow-light** [comment]
Yes that one will be added soon, once they release the USFM which is the format I need. They did so for the BSB so I expect it to happen sometime soon.

**Comment by kchristman54** [comment]
The [Majority Standard Bible](https://majoritybible.com/), produced by Berean, has the USFM and USFX formats over at [eBible](https://ebible.org/find/details.php?id=engmsb). Hopefully it can be added [paper.bible](https://paper.bible) soon.
